theano/sandbox/cuda/basic_ops.py

```diff
@@ -3354,7 +3354,7 @@ class GpuAlloc(GpuAllocEmpty):
     """
     __props__ = ('memset_0',)
 
-    def do_merge(self, node):
+    def do_merge(self, node1, node2):
         return True
```

Comment:
We should use the new default now, right? Otherwise nodes with different inputs will be merged together